content/docs/guides/usage-reporting.md

```diff
@@ -17,6 +17,17 @@ ks param set spartakus reportUsage false
 kubectl delete -n ${NAMESPACE} deploy spartakus-volunteer
 ```
 
+**To disable usage reporting** you need to delete spartakus component. 
+This command completely deletes any spartakus deployment, while the above 
+command only restarts spartakus with reportUsage set to false
+
+```
+ks delete default ${ENV} -c spartakus
+kubectl -n ${NAMESPACE} delete deploy -l app=spartakus
+ks component rm spartakus
+```
+
+
 To explictly enable usage reporting repeat the above steps setting reportUsage to `true`
```

Comment:
Thanks for moving the new text to above this position! Just one small change needs making as a result: the words "repeat the above steps" on line 20 don't work now. Since those words are not really necessary anyway, I suggest the following change.

On line 20, change this:
repeat the above steps setting reportUsage to `true`

to this:
set reportUsage to `true`